screens/BookDetail.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Button } from 'react-native';
+import styled from 'styled-components/native';
+const util = require('util'); 
```

Comment:
also remove this

Reply:
I remembered that you we're going to make the procedure with me, in order to "clone" the code in the MAC.
also the update file it's not in my pc either. In the meantime, I'll code in the oldest file.

Reply:
you can clone this repo and continue from this branch

Reply:
GitHub stands for it.
`git clone git@github.com:oskojess/ChallengeEntria.git`
`git checkout start-again`
that's all \o/

Reply:
Success!

Thanks <3